src/protocol/DocumentServer.js

```diff
@@ -37,15 +39,16 @@ define('polymer-designer/protocol/DocumentServer', [
       this.nodes = new Map();
       this.nextId = 1;
 
+      connection.on('command', this._onCommand.bind(this));
+      connection.on('getCaretPosition', this.getCaretPosition.bind(this));
```

Comment:
I wonder if there should be some sugar for the pattern of "map this event name to this method"